db/migrate/20170331041221_create_ctas.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCtas < ActiveRecord::Migration[5.0]
+  def change
+    create_table :ctas, id: :uuid do |t|
+      t.string :title
+      t.text :description
+      t.string :website
+      t.boolean :free, default: true
+      t.datetime :start_at
+      t.datetime :end_at
+      t.integer :action_type
+      t.uuid    :location_id
+      t.uuid    :contact_id
+      t.uuid    :call_script_id
+
+      t.timestamps
```

Comment:
Good call.

I've added that compound index.  I had to revise the criteria for uniqueness on the CTA model b/c Rails limits the length of the index name to 64 characters.   There are fewer fields we evaluate, but I think they're sufficient to exclude any potential duplicates.